common/lib/driver-definitions/src/storage.ts

```diff
@@ -290,16 +290,22 @@ export interface IDocumentServiceFactory {
     /**
      * Returns an instance of IDocumentService
      */
-    createDocumentService(resolvedUrl: IResolvedUrl, logger?: ITelemetryBaseLogger): Promise<IDocumentService>;
+     createDocumentService(resolvedUrl: IResolvedUrl, logger?: ITelemetryBaseLogger): Promise<IDocumentService>;
 
     /**
-     * Creates a new document with the provided options. Returns the document service.
-     * @param createNewSummary - Summary used to create file. If undefined, an empty file will be created and a summary
-     * should be posted later, before connecting to ordering service.
+     * Creates a new document with the provided options.
+     * @param request - Request required to create a new document.
+     * @param urlResolver - URL resolver required to resolve document URLs.
+     * @param createNewSummary - Optional. Summary used to create file.
+     * If undefined, an empty file will be created and a summary should be posted later,
+     * before connecting to ordering service.
+     * @param logger - Optional. Telemetry logger.
+     * @returns Returns the document service connected to the delta stream.
      */
     createContainer(
-        createNewSummary: ISummaryTree | undefined,
```

Comment:
this change cannot be made. ideally it's not necessary, but if it is you need to add a new optional method and somehow support both